chat-plugins/wifi.js

```diff
@@ -149,10 +149,10 @@ var LotteryGiveAway = (function () {
 		this.prize = options.prize;
 		this.maxwinners = options.maxwinners;
 		this.joined = {}; // userid: 0
-
+		var firstPart = this.prize[0];
 		this.reminder = '<center><div class = "broadcast-blue"><font size = 3><b>It\'s giveaway time!</b></font><br/>' +
 			'<font size = 1>Giveaway started by ' + Tools.escapeHTML(host.name) + '</font><br/><br/>' +
-			'<b>' + Tools.escapeHTML(giver.name) + '</b> will be giving away a <b>' + Tools.escapeHTML(this.prize) + '</b>!<br/>' +
+			'<b>' + Tools.escapeHTML(giver.name) + '</b> will be giving away ' + (Number(firstPart) && firstPart !== '1' ? '' : 'a') + ' <b>' + Tools.escapeHTML(this.prize) + '</b>!<br/>' +
```

Comment:
Code talk now. The way it's formatted, if someone decides to give away something like "4 Zekrom", you will have two spaces in the message:
> SILVERTACTIC will be giving away (space)(another space!) **4 Zekrom**!

To fix, trim the space before the `<b>` tag after your check for the number of Pokemon there are, and then use the string `'a '` (notice the space) instead of `'a'`.